cli.go

```diff
@@ -135,18 +136,31 @@ func fetchAuthURL(c *api.Client, role, mount, port string, callbackHost string)
 	return authURL, nil
 }
 
+// isWsl tests if the binary is being run in Windows Subsystem for Linux
```

Comment:
nit: s/isWsl/isWSL   [(Initialisms)](https://github.com/golang/go/wiki/CodeReviewComments#initialisms)

Reply:
ah, good to know, thanks!